utils/manage_events.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing, please email
+# opensource@seagate.com or cortx-questions@seagate.com.
+
+
+import logging
+from typing import List
+
+from cortx.utils.message_bus import MessageBus, MessageConsumer
+from ha.core.event_manager.event_manager import EventManager
+
+COMPONENT_ID = 'hare'
+
+class EventListener():
+    """This is listener for event manager
+
+    Typical use case of this class is as follows
+    subscribe(event list) - subscribe to events for listening, returns topic
+                            which can be used for listening
+    _listen(topic) - reading messages received on above topic
+    unsubscribe(event list) - unsubscribe as we dont want to receive these
+                              events anymore
+    get_message_type() - returns topic just in case user is not maintaining
+                         output of 'subscribe'
+    Note: 1. There is only 1 topic(message type, queue) for given component.
+          2. When component subcribes for event for the 1st time topic is
+             created and returned to user.
+          3. This topic will be used by user to listen for events.
+          4. After event is received in queue, user will read it, process it
+             and acknowledge it
+
+    """
+
+    event_manager = None
+    consumer = None
+
+    def __init__(self):
+        logging.debug('Inside EventListener')
+        self.event_manager = EventManager.get_instance()
+
+    def subscribe(self, event_list: List[str]) -> str:
+        logging.info('Subscribing for events: (%s)', str(event_list)[1:-1])
```

Comment:
Why do you make such a slice of a string?

Reply:
It is to make message simpler
`Subscribing for events: ('bgdelete')`
Otherwise it will look like
`Subscribing for events: (['bgdelete'])`

Reply:
I doubt that it makes sense. The readability is rather bad while the result is hardly beneficial.

Suggestion: 
```suggestion
        logging.info('Subscribing to events: %s', event_list)
```

Just FYI (I don't force you to apply this in this patch; this code will probably be changed in the future anyway):

The fact is that we use patterns in logging (e.g. `logging.debug('My value: %s', some_function())` to make the evaluations lazy. So in this example `some_function()` will be evaluated if and only if the log lever requires this log message to be rendered. If the log level is INFO or WARN, this unnecessary evaluation will even not happen.

Another point is that internally `%s` causes `str()` to happen just automatically. And that `str()` will also be lazy.
It means that `logging.info('Subscribing for events: (%s)', event_list)` will caluse `str(event_list)` lazy evaluation, but `logging.info('Subscribing for events: (%s)', str(event_list))` will force `str()` even if the log message is not required.

So it is just a good habit - keep logging lazy when possible.

Reply:
Done